### DISCUSSION TITLE
Enhancement: Layout Designer - Masonry-Style Layout

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#8691] [Ideas]
It would be nice if the layout designer could make use of a "masonry-style" layout for the field groups to minimise the need to scroll the page to access specific fields.

Maybe even an option to toggle collapse individual field group would also improve usability and the ability to view the relevent fields/field groups above the browser fold.

**Comment by brandonkelly** [comment]
Related to #913
